src/main/app/fees/feesClient.ts

```diff
@@ -23,8 +24,8 @@ export class FeesClient {
    * @returns {Promise.<number>} promise containing the fee amount in pounds
    */
   static async calculateIssueFee (claimValue: number): Promise<number> {
-    const fee: CalculationOutcome = await this.calculateFee(issueFeeCode, claimValue)
-    return MoneyConverter.convertPenniesToPounds(fee.amount)
+    return this.calculateFee(issueFeeEvent, claimValue, onlineChannel)
```

Comment:
Please also mention in the JSDoc that online channel is being used so there is no surprise when somebody tries to use this method to calculate fee for paper channel.